sdk/python/tests/unit/test_feature_service.py

```diff
@@ -55,3 +57,20 @@ def test_hash():
 
     s4 = {feature_service_1, feature_service_2, feature_service_3, feature_service_4}
     assert len(s4) == 3
+
+
+def test_feature_view_kw_args_warning():
```

Comment:
Add a test for the kwargs version not raising any deprecation warnings?

Reply:
Done